packages/vm/lib/evm/evm.ts

```diff
@@ -272,6 +273,10 @@ export default class EVM {
       }
     }
 
+    if (this._vm._common.eips().includes(2929)) {
+      this._state.addWarmAddress(message.to.buf)
```

Comment:
Note: this is in the wrong place, we should do this before the checkpoint in the new message (in case that the creation fails, then the address should /still/ be in the warm list)

Reply:
There will likely be a reason why you nevertheless put this here along with giving a comment, but it is not obvious too me (too difficult for some reasons?). Could you clarify?

Reply:
Yeah sorry this is a bit weird. I realized somewhere around pushing that this might be wrong, checked it, and realized that it is indeed wrong. It is more of a reminder comment that I should fix it :)

Reply:
😄